src/overlays/actors/ovl_En_Okarina_Tag/z_en_okarina_tag_cutscene_data.c

```diff
@@ -0,0 +1,137 @@
+#include "z_en_okarina_tag.h"
```

Comment:
I could be wrong, but I thought we wanted to include cutscene data files from within the main .c file instead of in the spec / cfg

Reply:
It depends whether it was included as a C file or linked separately in the original, since it's not always up to us.

Reply:
Ah, okay, makes sense

Reply:
Yea looks like it was linked separately here. The padding also doesnt line up unless I link it separately

Reply:
gorcha 👍